src/main/java/org/madhuri/app/message/util/HibernateUtil.java

```diff
@@ -0,0 +1,32 @@
+package org.madhuri.app.message.util;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+
+	
+	//private static SessionFactory sessionFactory;
+
+	
+
+	/*public HibernateUtil() {
+		Configuration configuration = new Configuration();
+		configuration.configure("hibernate.cfg.xml");
+
+		sessionFactory = configuration.buildSessionFactory();
+	}*/
+
+	public static Session getSession() {
```

Comment:
this creates a session for every getSession call ?
How can you optimize to only create a session once at the starting of the app ?
something like singleton pattern ?